### ISSUE TITLE
Incorrect typing for the events

### ISSUE THREAD

**Issue by Beramos** [m93a/svelte-plotly.js#11]
Hi, thx for maintaining this awesome wrapper.

I wonder if the typing of the event dispatches is correct. 

e.g. for the selected event: https://github.com/m93a/svelte-plotly.js/blob/91b1d12c49562b2aefeca81171a5529617681e43/src/lib/Plot.svelte#L109

should the type not be `CustomEvent<PlotSelectionEvent>` instead of `PlotSelectionEvent`

`PlotSelectionEvent` seems to be defined as

```typescript
export interface PlotSelectionEvent {
    points: PlotDatum[];
    range?: SelectionRange | undefined;
    lassoPoints?: SelectionRange | undefined;
}
```

in svelte when handling the on:selected dispatch, I still have to access the `PlotSelectionEvent using `e.detail.points`

```
<Plot
 on:selected={(e) => console.log(e.detail.points}
/>
```

Hence, I can't get the typing right the function handling the dispatch (replaced by the logging statement) expects type `CustomEvent<PlotSelectionEvent>` while  on:selected is typed as `PlotSelectionEvent`

**Comment by cshaa** [comment]
Hey Bram!
Thank you for the report! You are, of course, right – since I dispatch the event via Svelte's `createEventDispatcher`, they are all wrapped in a `CustomEvent`. I've tried to make a quick fix, but I wasn't able to test it thoroughly yet, so I published it as a [release candidate](https://www.npmjs.com/package/svelte-plotly.js?activeTab=versions). You can try it with `npm install svelte-plotly.js@1.1.0-rc1`. Please let me know if this fixes the issue ~

**Comment by Beramos** [comment]
Awesome! I'm quite new to typescript+svelte so I was not 100% sure ;D I tested the release candidate and it works for my case :+1:

**Comment by cshaa** [comment]
Published as 1.1.0 now
